wasp/apps/alarm.py

```diff
@@ -86,21 +87,29 @@ def __init__(self):
         self.alarms = (bytearray(3), bytearray(3), bytearray(3), bytearray(3))
         self.pending_alarms = array.array('d', [0.0, 0.0, 0.0, 0.0])
 
-        # Set a nice default
-        self.num_alarms = 1
-        for alarm in self.alarms:
-            alarm[_HOUR_IDX] = 8
-            alarm[_MIN_IDX] = 0
-            alarm[_ENABLED_IDX] = 0
-        self.alarms[0][_ENABLED_IDX] = _WEEKDAYS
-
+        self.num_alarms = 0
+        try:
+            with open("alarms.txt", "r") as f:
+                alarms = f.readlines()[0].split(";")
+            if "" in alarms:
+                alarms.remove("")
+            for alarm in alarms:
+                n = self.num_alarms
+                h, m, st = map(int, alarm.split(","))
+                self.alarms[n][0] = h
+                self.alarms[n][1] = m
+                self.alarms[n][2] = st
+                self.num_alarms += 1
+        except Exception:
+            pass
```

Comment:
I originaly removed the defaults because it overwrote the alarms in case of read/write failure when I was prototyping.

I ended up leaving it removed because I don't think it's a particularly good idea to have a default alarm. Your call :)

Reply:
I don't have any strong opinion about this, I think it's fine either way. Just thought you forgot to add it back